src/validations/profile.ts

```diff
@@ -59,8 +66,59 @@ export const wearableUrns: Validation = validationAfterADR75({
   }
 })
 
+export const profileEmotes: Validation = validationAfterADR74({
```

Comment:
I can't comment the corresponding line (61). Can we modify the error message to something that explicitly says it's a wearable: 
```typescript
`Each profile wearable should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid wearable pointer: (${pointer})`
```

Reply:
added